app/models/user.rb

```diff
@@ -7,4 +7,9 @@ class User < ApplicationRecord
   validates :email, presence: true, length: {maximum: Settings.max_email_length},
     format: {with: VALID_EMAIL_REGEX}, uniqueness: {case_sensitive: false}
   validates :password, presence: true, length: {minimum: Settings.min_password_length}
+
+  def User.digest string
```

Comment:
As there are several ways to declare class method in Ruby, you should declare following way according to the style guide
```
class << self 
  def method_name
    # method body
  end 
end 
```